lib/ClangImporter/ImportType.cpp

```diff
@@ -1183,6 +1183,10 @@ namespace {
             }
           }
 
+          if (bridgedType->isConstraintType() &&
```

Comment:
Can `bridgedType` really happen to be an protocol/composition here, or was this a forethought?

Reply:
Yes, it can be `Error`